wikimedia/wikipedia-ios
# [Image recs] Add basic Image Recommendations API calls

**Phabricator:**  https://phabricator.wikimedia.org/T355321

### Notes
* This PR adds the basic calls to retrieve image recommendation tasks and the recommendations per article

### Test Steps
1. Cherry-pick this commit to perform the calls on the explore feed e2134195404fd4e025ba7d7354915314ec05bba4
2. Make sure you either see the objects on the debug prints or you can inspect the calls on a proxy debugger
